pki/src/test/java/org/hyperledger/besu/pki/keystore/SoftwareKeyStoreWrapperTest.java

```diff
@@ -188,10 +189,10 @@ public void loadKeyStoreFromFile() {
             "validator",
             null);
 
-    assertThat(loadedKeyStore.getPublicKey("validator")).isNotNull();
-    assertThat(loadedKeyStore.getPrivateKey("validator")).isNotNull();
-    assertThat(loadedKeyStore.getCertificate("validator")).isNotNull();
+    assertNotNull(loadedKeyStore.getPublicKey("validator"));
+    assertNotNull(loadedKeyStore.getPrivateKey("validator"));
+    assertNotNull(loadedKeyStore.getCertificate("validator"));
     // CA -> INTERCA -> PARTNERACA -> VALIDATOR
-    assertThat(loadedKeyStore.getCertificateChain("validator")).hasSize(4);
+    assertEquals(loadedKeyStore.getCertificateChain("validator"), 4);
```

Comment:
This should be using hasSize or similar so we check the array size and not do a object comparison

Reply:
fixed